src/main/java/com/dft/veracross/model/students/studentWrapper/StudentInfo.java

```diff
@@ -0,0 +1,18 @@
+package com.dft.veracross.model.students.studentWrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
Do you need this annotation for this class ?

Reply:
> Do you need this annotation for this class ?

No, i removed.